lib/sprockets/sassc_processor.rb

```diff
@@ -0,0 +1,64 @@
+require 'sprockets/sass_processor'
+require 'base64'
+
+module Sprockets
+  class SasscProcessor < SassProcessor
+    def initialize(options = {}, &block)
+      # SassC engine does not currently support caching
+      # @cache_version = options[:cache_version]
+      # @cache_key = "#{self.class.name}:#{VERSION}:#{Autoload::SassC::VERSION}:#{@cache_version}".freeze
```

Comment:
`cache_key` is needed for sprockets own caching even if there is no specific caching in the processor. This way if the processor lib upgrades the sprockets cache is invalidated. So we need `@cache_key` but we don't need `@cache_version` only its value in the end of the cache key.